### ISSUE TITLE
How to build or integrate a release react native ios/android APP?

### ISSUE THREAD

**Issue by Resulte** [wuba/Taro-Mortgage-Calculator#47]
I want to build a release react native ios/android APP, and publish it to App Store. Have any tutorial or docs for this?

**Comment by zhiqingchen** [comment]
https://docs.taro.zone/docs/react-native

**Comment by Resulte** [comment]
I mean have any tutorial or docs to build a release App for this application(Taro-Mortgage-Calculator) in detail?

**Comment by zhiqingchen** [comment]
No, the project only packages jsbundle and required static resources. The process of packaging into apk or ipa is similar to RN.

The native code can refer to [taro-native-shell](https://github.com/NervJS/taro-native-shell).

Android packaging process can refer to https://github.com/NervJS/taro-native-shell/blob/0.69.0/.github/workflows/assemble_android_release.yml

iOS packaging process can refer to https://github.com/NervJS/taro-native-shell/blob/0.69.0/.github/workflows/assemble_ios_release.yml

What you need to do is to modify the [configuration](https://github.com/wuba/Taro-Mortgage-Calculator/blob/master/config/index.js#L81-L84) and change the output path of yarn build:rn to the corresponding location of the taro-native-shell.
